woocommerce/woocommerce-ios
# Shipping Labels: Move action sheet to individual package items to fix issue displaying popovers on iPad

Fixes #5211 

# Description
Currently we're showing a single action sheet on the Package Details screen, which works fine on iPhone but breaks on iPad since action sheet is displayed as popover on bigger screens and popovers require correct anchor points to look good.

To make sure each popover points to the correct button, we need to display them on each item row, so this PR handles a lot of changes:
- Adds unique ID for `ShippingLabelPackageItem`, which is used for item row ID - to identify the correct row that's moved (instead of based on product ID, which is not unique in a package).
- Removes action sheet from package list.
- Moves "Move" button from single package to item row. Adds action sheet to item row.
- Updates single package view model:
  - Adds unique ID for the view model using ID of the associating package for easier identifying.
  - Adds package number needed to display on action sheet.
  - Adds method to update list of action sheet buttons for each package item.
  - Adds action sheet info when creating item rows.

Several workarounds to be aware of:
- SwiftUI doesn't allow displaying more than one popover on the same screen, so before displaying one, I have to trigger a callback to parent view to dismiss any presented one.
- The action buttons need other packages' names to display options to move to other packages, so action buttons cannot be set in initializer of a single package view model. I have to update these buttons as an extra step after creating the view models. 

# Demo
https://user-images.githubusercontent.com/5533851/137424774-06d322c1-914f-429d-afbd-63296f5631ed.mp4

# Testing
1. Make sure your test store has configured WCShip plugin.
2. Create an order with multiple items.
3. Build the app to an iPad. Select the order and tap Create Shipping Label.
4. Confirm Ship From and Ship To address.
5. On Package Details screen, tap Move button on any item. Notice that the popover points to the correct button. Make sure that the popover contains all options available and each option works correctly.
6. With a popover present, try tapping Move button on any other item of the same or different package. Make sure that no glitch happens.
7. Build the app again on an iPhone, make sure that everything still works properly.

Update release notes:
